src/main/java/org/terasology/web/serverAdminManagement/PermissionType.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.web.serverAdminManagement;
+
+import org.terasology.logic.permission.PermissionManager;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This enum is used for determining admin permissions.
+ */
+public enum PermissionType {
+    NO_PERMISSION,
+    CONSOLE_CHEAT,
+    CONSOLE_USER_MANAGEMENT,
+    CONSOLE_SERVER_MANAGEMENT,
+    CONSOLE_DEBUG,
+    INSTALL_MODULES,
+    CREATE_BACKUP_RENAME_GAMES,
+    DELETE_GAMES,
+    START_STOP_GAMES,
+    CHANGE_SETTINGS,
+    ADMIN_MANAGEMENT;
+
+    public static Map<PermissionType, String> getConsolePermissionsMap() {
```

Comment:
I think using a map is a good idea. An alternative IMHO could be storing the mapped console permission inside a field of the instance of the enum, e.g. having a `private String mappedConsolePermission;` (remember that an enum is just like another class, it just has a fixed number of instances - the enum entries; see for example the [planets example here](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html)), or maybe it could be an `Optional<String>` since some entires have no mapped console permission. Then you would assign it in the constructor, and you could write the entries like `CONSOLE_CHEAT(PermissionManager.CHEAT_PERMISSION), START_STOP_GAMES(Optional.empty())` (or `null` if you don't use `Optional`, or make an overloaded constructor which takes no arguments for the values which have no mapping, etc - various possibilities).

Then you would be able to know the mapped console permission via a, for example, `getMappedConsolePermission()` method.

However I see that you call `getConsolePermissionsMap()` just once and you need the mappings for all of them at once - so I'd say your way is equally good. Just take it as a hint for similar situations if you prefer keeping it this way